specs/altair/sync-protocol.md

```diff
@@ -256,3 +351,395 @@ def process_light_client_update(store: LightClientStore,
+    assert header.slot == state.slot
+    period = compute_sync_committee_period(compute_epoch_at_slot(header.slot))
+
+    attested_header = attested_state.latest_block_header.copy()
+    attested_header.state_root = hash_tree_root(attested_state)
+    assert hash_tree_root(attested_header) == state.latest_block_header.parent_root
+    assert attested_header.slot == attested_state.slot
+    attested_period = compute_sync_committee_period(compute_epoch_at_slot(attested_header.slot))
+
+    if period == attested_period and hasattr(attested_state, "next_sync_committee"):
+        optimistic_update = OptimisticLightClientUpdate(
+            attested_header=attested_header,
+            sync_aggregate=block.message.body.sync_aggregate,
+            fork_version=state.fork.current_version,
+        )
+
+    return optimistic_update
+```
+
+## Light Client
+
+This section describes the expected actions of a light client in the Ethereum proof-of-stake protocol.
+
+### Sync via libp2p
+
+1. The light client MUST be configured with a spec/preset,
+   and MUST be provided with a trusted checkpoint block root out-of-band.
+   The trusted block's slot SHOULD fall into the weak subjectivity period, i.e., it SHOULD NOT be stale.
+
+2. The light client browses for libp2p peers and connects to them.
+   Upon connection to a remote peer, it sends a [`Status`](../phase0/p2p-interface.md#status) request.
+
+3. A [`GetLightClientBootstrap`](#getlightclientbootstrap) request is sent for the trusted block root
+   to fetch a `bootstrap` object of type `LightClientBootstrap`.
+
+4. The received `bootstrap` object is passed to [`initialize_light_client_store`](#initialize_light_client_store) to obtain a local `store` object of type [`LightClientStore`](#lightclientstore).
+   If `bootstrap.header.slot` is not within the weak subjectivity period, the sync SHOULD fail.
+
+5. The local clock is started and [`process_slot_for_light_client_store`](#process_slot_for_light_client_store) is called on every slot.
+   The `store`'s sync committee period and the current sync committee period are computed
+   from `store.finalized_header.slot` and `current_slot`.
+
+6. [`LightClientUpdatesByRange`](#lightclientupdatesbyrange) requests are sent for sync committee period range
+   `[store_sync_committee_period, current_sync_committee_period]` (both inclusive).
+   The [`LightClientUpdate`](#lightclientupdate) responses are passed to `process_light_client_update` in ascending order.
+   If, for any sync committee period before `current_sync_committee_period`,
+   a `LightClientUpdate` cannot be obtained , the sync fails.
+
+7. Once the `store_sync_committee_period` initially reaches `current_sync_committee_period`,
+   or after syncing through multiple sync committee periods when re-joining the network,
+   `store.previous_max_active_participants` and `store.current_max_active_participants`
+   MUST be reset to `0` to clear the `store`'s participation tracker.
```

Comment:
Related PR: https://github.com/ethereum/consensus-specs/pull/2810
With that PR, this step is no longer needed.